doc/api/net.md

```diff
@@ -544,40 +729,41 @@ If `data` is specified, it is equivalent to calling
 added: v0.9.6
 -->
 
-The string representation of the local IP address the remote client is
-connecting on. For example, if you are listening on `'0.0.0.0'` and the
-client connects on `'192.168.1.1'`, the value would be `'192.168.1.1'`.
+Provides the string representation of the local IP address the remote peer is
+connecting on. For example, if the `net.Socket` is listening on `'0.0.0.0'` and
+the peer connects on `'192.168.1.1'`, the value would be `'192.168.1.1'`.
 
 ### socket.localPort
 <!-- YAML
 added: v0.9.6
 -->
 
-The numeric representation of the local port. For example,
-`80` or `21`.
+Provides the numeric representation of the local port. For example, `80` or
+`21`.
```

Comment:
"a numeric representation of a number" is what this is effectiely saying, how about `The local port number.`? what if its still unbound, and what if its a local domain socket?